web_widget_slick/README.rst

```diff
@@ -0,0 +1,95 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+=====================
+Slick Carousel Widget
+=====================
+
+This module provides a Slick Carousel widget for use in the Odoo backend web interface.
+
+Usage
+=====
+
+Default usage is on a One2many attachment field, as defined below::
+
+    class SlickExample(models.Model):
+        _name = 'slick.example'
+        _description = 'Slick Example Model'
+        image_ids = fields.One2many(
+            name='Images',
+            comodel_name='ir.attachment',
+            inverse_name='res_id',
+        )
+
+Assuming the above model, you would add a Slick Carousel on the
+``image_ids`` column by using the following field definition in the
+model's form view::
+
+    <field name="image_ids" widget="one2many_slick_images" options="{}"/>
+
+Options
+-------
+
+The widget passes options directly through to Slick, so any `setting
+available to Slick`_ is available to the widget. Additional options
+specific to Odoo are:
+
++-----------------+--------------+---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Name            | Type         | Default             | Description                                                                                                                                                                 |
++=================+==============+=====================+=============================================================================================================================================================================+
+| ``fieldName``   | ``String``   | ``datas``           | Field to lookup on relation table. Defaults to ``datas``, which is the data field used in ``ir.attachment`` table. This would be used to define a custom attachment model   |
++-----------------+--------------+---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``modelName``   | ``String``   | ``ir.attachment``   | Model of attachment relation. This would be used to define a custom attachment model instead of default ``ir.attachment``                                                   |
++-----------------+--------------+---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+.. _setting available to Slick: http://kenwheeler.github.io/slick/#settings
+
+Example Module
+--------------
+
+An example implementation, for instructional purposes as well as convenient
```

Comment:
How does a functional reviewer find the menu for testing? Assume that a functional reviewer does not review code, so pointing them to code is no bueno